dashboard/consts.py

```diff
@@ -79,3 +79,6 @@
     "dashboard",
     "gcs",
 }
+PARENT_HEALTH_CHECK_BY_PIPE = env_bool(
+    "RAY_enable_pipe_based_agent_to_parent_health_check", False
```

Comment:
The default value is true on the c++ side. Is there a way to read c++ config in Python?

Reply:
there's no way. I think it is okay, (it is tested anyway). if you want, I can change it to true, but it will be in sync anyway because the env var is always passed